packages/perseus/src/widgets/interactive-graphs/graph-locked-labels-layer.tsx

```diff
@@ -19,11 +19,14 @@ export default function GraphLockedLabelsLayer(props: Props) {
         }
 
         if (
-            flags?.["mafs"]?.["locked-point-labels"] &&
-            figure.type === "point"
+            // Point flag + point type
+            (flags?.["mafs"]?.["locked-point-labels"] &&
+                figure.type === "point") ||
+            // Line flag + line type
+            (flags?.["mafs"]?.["locked-line-labels"] && figure.type === "line")
         ) {
             return (
-                <React.Fragment key={`point-${i}`}>
+                <React.Fragment key={`locked-figure-${i}`}>
```

Comment:
I think it would be okay for the key to just be `i` here. Keys only have to be unique within the current array of elements.